test/E2ETest/Microsoft.Test.E2E.AspNet.OData/QueryComposition/SelectExpandTests.cs

```diff
@@ -31,13 +28,12 @@ public SelectExpandTests(WebHostTestFixture fixture)
         {
         }
 
+        private static int SelectCustomerPropertyCount =>
```

Comment:
Any special reason that lambda op is needed here?

Reply:
This isn't a lambda; It's an expression-bodied method (a feature of C# 6).

Reply:
@brjohnstmsft why not just using an simple assignment "="?

Reply:
Good point; Updated